Automattic/jetpack
# Jetpack Boost: Improve Image CDN Quality settings UI

## Proposed changes:
- Isolated global variables and removed them from Image CDN Quality settings.
- Updated UI for the quality slider in Image CDN settings.
- Lowered opacity for disabled elements for better visual distinction.
- Re-aligned headers for better readability and consistency.
- Minor code formatting and cleanup.

![2023_09_269eJaTSPi 13](https://github.com/Automattic/jetpack/assets/988095/d32ff1bc-d0bf-4fea-b73f-36b3f5f616de)

![2023_09_26Ru24M4lg 12](https://github.com/Automattic/jetpack/assets/988095/b19a8cb1-c9ec-4af8-9ae2-78300382b4bd)

### Other information:



## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
N/A

## Testing instructions:
- Check the Image CDN settings page to observe updated UI for quality sliders.
- Test the responsiveness of the UI on different screen sizes.
- Verify that disabled UI elements visually appear different (lower opacity).
- Check for consistent alignment of headers across the plugin.
